### PR TITLE
Update all namespaces to minimal namespaces

### PR DESCRIPTION
**Pull request by Kayanski**
This PR aims at providing variables to all storage keys to minimize them without loosing data.

Also moved `ica-client` state to satisfy the same standard as the other contracts

### Checklist

- [ ] CI is green.
- [x] Changelog updated.

### CODE REVIEW

**Review by Buckram123 — COMMENTED** [review]
LGTM for native contracts. `common_namespace.rs` definitely needs discussion and refactor

File: framework/packages/abstract-std/src/objects/common_namespace.rs [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
-pub const BASE_STATE: &str = "base_state";
+pub const BASE_STATE: &str = "a";
 /// namespace for contract Admin
-pub const ADMIN_NAMESPACE: &str = "admin";
+pub const ADMIN_NAMESPACE: &str = "b";
 /// storage key for cw_ownable::Ownership
-pub const OWNERSHIP_STORAGE_KEY: &str = "ownership";
+pub const OWNERSHIP_STORAGE_KEY: &str = "c";
+/// storage key for Account Id
+pub const ACCOUNT_ID_STORAGE_KEY: &str = "d";
```
Comment (Buckram123) [inline framework/packages/abstract-std/src/objects/common_namespace.rs:8]: Let's have it as "id"? It has same key as `ACCOUNT_MODULES` for account right now

Comment (Kayanski) [reply framework/packages/abstract-std/src/objects/common_namespace.rs]: This was modified in develop/v2 to be included inside account, so this is now a letter in the account namespace

File: framework/packages/abstract-std/src/objects/common_namespace.rs [hunk 2]
```diff
@@ -1,6 +1,10 @@
 /// namespace key for State Item
-pub const BASE_STATE: &str = "base_state";
+pub const BASE_STATE: &str = "a";
```
[the remark this answers is not in the document]
Comment (Kayanski) [reply framework/packages/abstract-std/src/objects/common_namespace.rs]: reverting for discussion

File: framework/packages/abstract-std/src/objects/common_namespace.rs [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
 /// namespace for contract Admin
-pub const ADMIN_NAMESPACE: &str = "admin";
+pub const ADMIN_NAMESPACE: &str = "b";
 /// storage key for cw_ownable::Ownership
-pub const OWNERSHIP_STORAGE_KEY: &str = "ownership";
+pub const OWNERSHIP_STORAGE_KEY: &str = "c";
```
[the remark this answers is not in the document]
Comment (Kayanski) [reply framework/packages/abstract-std/src/objects/common_namespace.rs]: Reverted for now

[the remark this answers is not in the document]
Comment (CyberHoward) [reply framework/packages/abstract-std/src/objects/common_namespace.rs]: > Are we fine breaking standards here? @CyberHoward

No, we should stick to the defined key for ownership afaik.

[the remark this answers is not in the document]
Comment (CyberHoward) [reply framework/packages/abstract-std/src/objects/common_namespace.rs]: Same for cw2

File: framework/packages/abstract-std/src/objects/common_namespace.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (CyberHoward) [reply framework/packages/abstract-std/src/objects/common_namespace.rs]: > Don't we want to keep something more unique here, as we don't control what users will put in the modules state?

Yup think `base_state` makes more sense here as well. Don't want to run the risk of conflicting with user-defined ID.

**Comment by CyberHoward** [comment]
> What if we keep common namespace as something that would make sense, instead of having 1 byte keys for all of the state:
> 
> * Admin as "admin" (standard)
> * Governance as "ownership" (standard)
> * Config as "cfg"
> * AccountId as "id" or "ai"
> * BaseState as "bs"

This also makes sense but then it needs to be clearly documented on the state to ensure that devs don't conflict with the shorter namespaces.

File: framework/packages/abstract-std/src/objects/storage_namespaces.rs
Comment (Buckram123) [inline framework/packages/abstract-std/src/objects/storage_namespaces.rs:1]: Didn't we want to have CONFIG always as "cfg"?

**Review by Buckram123 — APPROVED** [review]
LGTM, what do you think about config key?

File: framework/packages/abstract-std/src/objects/storage_namespaces.rs
Comment (Kayanski) [reply framework/packages/abstract-std/src/objects/storage_namespaces.rs:1]: changed

**Review by CyberHoward — APPROVED** [review]

**Pull request merged**